css/src/tokens/colors.scss

```diff
@@ -66,8 +67,10 @@ $score-high: var(--theme-score-high);
 
 $primary: var(--theme-primary-base);
 $primary-background: var(--theme-primary-background);
+$primary-background-hover: var(--theme-primary-background-hover);
 $primary-background-glow-high-contrast: var(--theme-primary-background-glow-high-contrast);
 $primary-dark: var(--theme-primary-dark);
+$primary-dark-hover: var(--theme-primary-dark-hover);
```

Comment:
Similar question as below, do we need the -dark-hover and background-hover css variables created in this file?

Reply:
Same as above